packages/web-api-client/src/a11y-service-credential.ts

```diff
@@ -40,4 +33,28 @@ export class A11yServiceCredential {
 
         return request.defaults(authOptions);
     }
+
+    private async getTokenWithRetries(numRetries: number): Promise<TokenResponse> {
```

Comment:
I'm a big fan of recursion but I always try to go for iteration for the sake of performance. 
I know in this case the data set or input is small so the difference maybe insignificant, but usually I go for recursion for readability and maintainability, even I believe iteration would be more readable here, so if I were you I'd go for iteration here.
You don't have to change it if you don't want to since as I mentioned it's insignificant in this case but I it just wanted to share my mind.